src/sentry/api/event_search.py

```diff
@@ -684,6 +684,12 @@ def to_list(value):
     # Long term these will become more complex functions but these are
     # field aliases.
     "apdex": {"result_type": "number", "aggregations": [["apdex(duration, 300)", "", "apdex"]]},
+    "impact": {
+        "result_type": "number",
+        "aggregations": [
+            ["(1 - apdex(duration, 300) + (1 - 1 / sqrt(uniq(user))) * 3", "", "impact"]
```

Comment:
Looks like you're missing a closing `)` bracket somewhere in this expression.

Reply:
good catch